writing-visualising/dashboards.qmd

```diff
@@ -513,6 +513,39 @@ An example of a basic shiny test to check that your app loads up correctly can b
 
 ---
 
+#### Migration to shinytest2
+
+---
+
+The shinytest package has been superceded by the [shinytest2 package](https://rstudio.github.io/shinytest2/). If you already have shinytest UI tests in place for your dashboard, there is a function available to migrate these tests for you so that you do not have to rewrite them all yourself. 
+
+There is [guidance on migrating to shinytest2](https://rstudio.github.io/shinytest2/articles/z-migration.html) available. To complete the migration, run `shinytest2::migrate_from_shinytest(PATH_TO_APP)`, making sure to include the full filepath. The function will search for your "tests" folder, and will then look inside the folder for any scripts containing tests. A new `testthat` folder will be automatically created, containing the code that runs your tests, as well as the snapshots generated when your tests are run. 
+
+There are a couple of potential issues that you may run into:
+
+**Chromote**
+
+Shinytest2 makes use of the Chromote package, which needs to connect to your local instance of Google Chrome. Sometimes your copy of Chrome might be installed in a different place to where Chromote expects to find it, which means that your tests will not be able to run. There is information on how to resolve this issue [in the Tips for Using R section](../learning-development/r.html#shinytest2---invalid-path-to-chrome-error-message)
+
+**Formatting of line endings**
+
+The migration function will only work if specific line endings are used in your code. If you receive an error that looks like this, then you may be having this issue:
+
+``` 
```

Comment:
I don't think this code block is formatting right? [image from local build on this branch]
![image](https://github.com/dfe-analytical-services/analysts-guide/assets/52536248/1f124f87-97bb-4630-a216-91a2c58c8b0c)